dev_tools/ownership_utils.py

```diff
@@ -29,21 +29,31 @@ def get_first_owner(self, file: Path) -> Optional[str]:
         return owners[0] if owners else None
 
     def get_owners(self, file: Path) -> Tuple[str, ...]:
+        file_relative = file.relative_to(self._repo_dir)
         for ownership in self._ownerships:
-            if self.is_file_covered_by_pattern(file.relative_to(self._repo_dir), ownership.pattern):
+            if self.is_file_covered_by_pattern(file_relative, ownership.pattern):
                 return ownership.owners
 
         return ()
 
+    @staticmethod
+    def is_path_prefix(path: str, prefix: str) -> bool:
+        """Check if `prefix` is one of the parents of `path`, including itself."""
+        if not path.startswith(prefix):
+            return False
+        path_length, prefix_length = len(path), len(prefix)
+        if path_length == prefix_length:
```

Comment:
```suggestion
        prefix_length = len(prefix)
        if len(path) == prefix_length:
```

To avoid multiple assignments in a single line